gnovm/cmd/gno/run.go

```diff
@@ -58,6 +60,20 @@ func (c *runCfg) RegisterFlags(fs *flag.FlagSet) {
 		"main()",
 		"value of expression to evaluate. Defaults to executing function main() with no args",
 	)
+
+	fs.BoolVar(
+		&c.debug,
+		"debug",
+		false,
+		"enable interactive debugger using stdin and stdout",
+	)
+
+	fs.StringVar(
+		&c.debugAddr,
+		"debug-addr",
```

Comment:
Just curious: where do you see the biggest use case for remote debugging?

Reply:
For me it's IDEs and VSCode extension. if we want to integrate debugger in the UI.

Reply:
The use-cases for remote debugging are:
- debug without interfering with standard input and output, if used by the target program
- possible integration with an IDE, using an extension, as mentioned by Hariom
- possibility to dynamically attach the debugger to an already started program, even in daemon mode. This will be added next.
- possibility to develop scriptable tracing and monitoring tools on top of the debugger.